util/dvsim/utils.py

```diff
@@ -186,17 +187,14 @@ def find_and_substitute_wildcards(sub_dict,
     return sub_dict
 
 
-def md_results_to_html(title, css_path, md_text):
+def md_results_to_html(title, css_file_path, md_text):
```

Comment:
```suggestion
def md_results_to_html(title, css_file, md_text):
```

Reply:
updated, thanks so much for your time and suggestions!